ui/packages/button/src/Button.svelte

```diff
@@ -39,6 +39,9 @@
 	}
 
 	button[disabled] {
+		opacity: 0.5; lowed;
+		opacity: 0.5; /* lowe
+		filter: grayscale(30%); ilter: grayscale(30%); /* app
 		cursor: not-allowed;
```

Comment:
Something we're has happened here. The css auto-ordering vscode extension doesn't play too nicely if there are comments and sometimes garbles the output like this.

Reply:
Ah interesting, the comments are unnecessary anyways. Will remove and rerun formatter